src/services/exportInfoMap.ts

```diff
@@ -327,7 +327,9 @@ namespace ts {
             const seenExports = new Map<Symbol, true>();
             const checker = program.getTypeChecker();
             const defaultInfo = getDefaultLikeExportInfo(moduleSymbol, checker, compilerOptions);
-            if (defaultInfo && !checker.isUndefinedSymbol(defaultInfo.symbol)) {
+            // Note: I think we shouldn't actually see resolved module symbols here, but weird merges
+            // can cause it to happen: see 'completionsImport_mergedReExport.ts'
+            if (defaultInfo && !checker.isUndefinedSymbol(defaultInfo.symbol) && defaultInfo.symbol.name[0] !== `"`) {

```

Comment:
instead of name not starting with `"`, can you instead check whether the symbol is an alias, or perhaps that none of the declarations are modules?

Reply:
`export const foo = 0` is not an alias, so no to the former. Checking that none of the declarations are _external_ modules is what I’m effectively doing. This is a really hot path so I want to do it as cheaply as possible, so I prefer not to look through declarations if there’s a faster way, but I guess this is imprecise, since a computed property name on an `export=`’d object could be quoted.

Later: there’s a util for that, which checks for both the `Module` symbol flag and the first character of the name—that works for me.